.kitchen.yml

```diff
@@ -1,5 +1,32 @@
 ---
+provisioner:
+  data_bags_path: test/integration/data_bags
+  encrypted_data_bag_secret_key_path: test/integration/encrypted_data_bag_secret
 suites:
   - name: default
     run_list:
+      - recipe[osl-imap-test::default]
       - recipe[osl-imap::default]
+  - name: auth_sql
+    run_list:
+      - recipe[osl-imap::default]
```

Comment:
How much work would it be to setup mysql here for test kitchen and do similar tests you do in the default suite but for SQL auth?

Reply:
The most recent commit tests SQL auth with a local database.